tests/end_to_end/http_server/large_files/src/list_files.ts

```diff
@@ -0,0 +1,194 @@
+import { id } from 'azle';
+import { readdir, stat } from 'fs/promises';
+import { basename, join } from 'path';
+
+export async function ls(
```

Comment:
We could look for a off the shelf library for this if we wanted

Reply:
I looked around and I couldn't find anything, I think it's simple enough that I'm not too worried about it

Reply:
Let's get this to work, there is a bug in the npm package maybe at the ic-wasi-polyfill level

Reply:
https://github.com/demergent-labs/azle/issues/1995